### PR TITLE
Fixes cleaning not awarding cleaning XP

### LINKED ISSUE #49383: Can't gain XP for scrubbing skill
What it says on the tin, you gain no experience from mopping etc.

### PR DESCRIPTION
**Pull request by Mickyan**
Fixes #49383

Janitors rejoice

:cl:
fix: Cleaning messes with the mop and soap now correctly awards cleaning experience
/:cl:

### CODE REVIEW

**Comment by Dennok** [comment]
You lost cleaning XP on cleaning masterpiece of art? (things with positive cleanies.beauty (Dont know it exists or no))

**Comment by TiviPlus** [comment]
Jannies mad PR

**Review by spookydonut — CHANGES_REQUESTED** [review]
why not just make `CLEAN_SKILL_BEAUTY_ADJUSTMENT` negative

**Comment by Mickyan** [comment]
that does make more sense uh

File: code/__DEFINES/skills.dm [hunk 1]
```diff
@@ -24,5 +24,5 @@
 #define GetSkillRef(A) (SSskills.all_skills[A])
 
 //number defines
-#define CLEAN_SKILL_BEAUTY_ADJUSTMENT	15//It's a denominator so no 0. Higher number = less cleaning xp per cleanable
+#define CLEAN_SKILL_BEAUTY_ADJUSTMENT	-15//It's a denominator so no 0. Higher number = less cleaning xp per cleanable. Negative value means cleanables with negative beauty give xp.
```
Comment (Dennok) [inline code/__DEFINES/skills.dm]: Сleanables with positive beauty withdraw xp.
You mop up Mona Lisa! Ignorant bastard!

Comment (Mickyan) [reply code/__DEFINES/skills.dm]: When calculating exp gains it sets the max to 0 so you don't lose experience, but you don't gain any either

Comment (Dennok) [reply code/__DEFINES/skills.dm]: Wrong. This
https://github.com/tgstation/tgstation/blob/cd9c3a8f85edec403d583f849c4af7bb533965d9/code/datums/mind.dm#L130
cap minimum of skill XP to 0

Comment (Mickyan) [reply code/__DEFINES/skills.dm]: I don't understand what you're saying, you don't lose XP from cleaning messes with positive beauty.

Comment (Dennok) [reply code/__DEFINES/skills.dm]: Ups, you right. Here
https://github.com/tgstation/tgstation/blob/0373a69ee974281dbbe904e426d9bcf71a053db1/code/game/objects/items/mop.dm#L33
clean xp capped to be positive.

**Comment by ExcessiveUseOfCobblestone** [comment]
Travis says pending in PR menu but it's successful when you drill down

**Pull request merged**